components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -765,6 +765,91 @@ void shoc_diag_second_moments_ubycond_f(Int shcol, Real* thl, Real* qw, Real* wt
   ekat::pack::device_to_host({thl, qw, qwthl, wthl, wqw, uw, vw, wtke}, shcol, host_views);
 }
 
+void compute_diag_third_shoc_moment_f(Int shcol, Int nlev, Int nlevi, Real* w_sec,
+                                      Real* thl_sec, Real* qw_sec, Real* qwthl_sec,
+                                      Real* wthl_sec, Real* tke, Real* dz_zt,
+                                      Real* dz_zi, Real* zt_grid, Real* zi_grid,
+                                      Real* isotropy_zi, Real* brunt_zi, Real* w_sec_zi,
+                                      Real* thetal_zi, Real* wthv_sec_zi, Real* w3)
+{
+  // These inputs are unused in the Fortran implementation
+  (void)qw_sec;
+  (void)qwthl_sec;
+  (void)zt_grid;
+  (void)zi_grid;
+  (void)wthv_sec_zi;
```

Comment:
Should these inputs be removed further up the call stack?

Reply:
Not used in either impl, right? Then just rm them from F90. That's BFB, so can do that in this PR.